shareaholic.libraries.yml

```diff
@@ -0,0 +1,21 @@
+main:
+  version: 1.x
+  css:
+    module:
+      assets/css/bootstrap.css: {}
+      assets/css/bootstrap-responsive.css: {}
+      assets/css/main.css: {}
+#  js:
+#    assets/js/bootsrap.js: {minified: false}
+#    assets/js/jquery_custom.js: {minified: false}
+#    assets/js/jquery_ui_custom.js: {minified: false}
+#    assets/js/main.js: {minified: false}
+
+
+tos-modal:
+  version: 1.x
+  css:
+    module:
+      assets/css/reveal.css: {}
```

Comment:
The ToS modal is pretty simple. I'm not sure we need to add Reveal just for that. Optional: replace with something much simpler. For example: https://www.w3schools.com/howto/howto_css_modals.asp

Context: reveal was previously used for the iframes. which are now gone.